platform-configs/protectli-vp6650.robot

```diff
@@ -0,0 +1,21 @@
+*** Settings ***
+Resource    include/protectli-vp66xx.robot
+
+
+*** Variables ***
+${SNIPEIT}=                     no
+${INITIAL_CPU_FREQUENCY}=       1100
+${DEF_CORES}=                   6
+${DEF_THREADS}=                 2
+${DEF_CPU}=                     12
+${DEF_ONLINE_CPU}=              0-11
+${DEF_SOCKETS}=                 1
+
+${POWER_CTRL}=                  sonoff
+${WIFI_CARD_UBUNTU}=            ${EMPTY}
+${LTE_CARD}=                    ${EMPTY}
+${DEVICE_NVME_DISK}=            Non-Volatile memory controller
+${USB_MODEL}=                   SanDisk
+
+${DMIDECODE_PRODUCT_NAME}=      VP6650
+${PLATFORM_SPI_CHIP}=           MX25L12805D
```

Comment:
Seems to me we need this parameter here, since we're using one generic keyword for two platforms with different SPI chips detected. Not sure why pre-commit considers this variable unused, when it is, in fact, used [right here](https://github.com/Dasharo/open-source-firmware-validation/pull/249/files#diff-954bbd10362981fae23372ee9d9f283260be3d63c909070ea207236051e7096eR59).